### PR TITLE
create server_configuraton

### PR DESCRIPTION
**Pull request by szaimen**
fix https://github.com/nextcloud/vm/issues/1223

### CODE REVIEW

**Comment by szaimen** [comment]
@enoch85 this is how I would do it.
Do you have already some thoughts on that?
It should already work but didn't test.

File: static/server_configuration.sh [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    *"Static IP"*)
+        clear
+        run_static_script static_ip
+        rm -f "$SCRIPTS"/lib.sh
+    ;;&
+    *"Automatic updates"*)
+        clear
+        run_static_script automatic_updates
+    ;;&
+        *"Activate TLS"*)
```
Comment (enoch85) [inline static/server_configuration.sh:41]: this needs better indentation

Comment (szaimen) [reply static/server_configuration.sh]: You are right. Is fixed.

**Comment by enoch85** [comment]
Looks good to me, but let's wait with merging until it's tested.

**Comment by szaimen** [comment]
BTW: do we need this here?
https://github.com/nextcloud/vm/blob/e5f07cc895ec557220d3977cc066bdb76502e541/static/static_ip.sh#L12-L32

**Comment by enoch85** [comment]
> BTW: do we need this here?

Yes, since it should run also without internet.

**Comment by szaimen** [comment]
> > BTW: do we need this here?
> 
> Yes, since it should run also without internet.

👍 
Would you add it?

**Comment by enoch85** [comment]
Tested and works as intended.

**Pull request merged**